opensrp-chw/src/lmh/assets/ec_client_fields.json

```diff
@@ -249,6 +249,28 @@
             "field": "baseEntityId"
           }
         },
+        {
+          "column_name": "thinkmd_id",
+          "type": "Client",
+          "json_mapping": {
+            "field": "identifiers.thinkmd_id"
+          }
+        },
+        {
+          "column_name": "html_assessment",
+          "type": "Event",
+          "json_mapping": {
+            "field": "obs.values",
+            "formSubmissionField": "generatedDiv"
+          }
+        },
+        {
+          "column_name": "care_plan_date",
+          "type": "Event",
+          "json_mapping": {
+            "field": "eventDate"
```

Comment:
This will always be changed by each event. Store a custom obs in thinkmd event and map it here